upgrade-cockroach.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Upgrade Your Cluster's Version of CockroachDB
```

Comment:
Similar to feedback on the sidenav, I think `Upgrade a Cluster` might be simpler and sufficient. But I'm fine with this as well.